.github/workflows/multi-agent-triage.yml

```
@@ -13,6 +13,10 @@ permissions:
   issues: write
   pull-requests: write
   id-token: write
+  workflows: write
```

actionlint (1.7.10)

unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)